python/tank/deploy/tank_commands/action_base.py

```diff
@@ -162,13 +162,15 @@ def _validate_parameters(self, parameters):
         
     def run_interactive(self, log, args):
         """
-        Run this API in interactive mode. This mode may prompt the user for input via stdin.
+        Run this API in interactive mode. 
```

Comment:
i was just doing a bunch of doc string cleanup, trying to make it a little bit better than it was.
The first line should be a summary, I think that's why i split it. A lot of these doc strings are not perfect, but at least they are better than they used to ! :)